src/views/sandwich-view.tsx

```diff
@@ -64,22 +141,42 @@ class SandwichView extends StatelessComponent<SandwichViewProps> {
           </div>
         </div>
       )
-    }
-
-    return (
-      <div className={css(commonStyle.hbox, commonStyle.fillY)}>
+    } else {
+      functionsView = (
         <div className={css(style.tableView)}>
           <ProfileTableViewContainer activeProfileState={this.props.activeProfileState} />
         </div>
+      )
+    }
+
+    container = (
+      <div
+        className={css(commonStyle.hbox, commonStyle.fillY, style.flexDisplay)}
+        onMouseUp={e => this.stopResize(e)}
```

Comment:
I think this is covered by the binding of `mouseup` on `window`, so I think this is unnecessary.